RHTE-LAB.adoc

```diff
@@ -0,0 +1,844 @@
+service/eunomia-operator created
+deployment.apps/eunomia-operator created
+--------------------------------------------------------------------------------
+
+. Check Eunomia operator is running:
++
+[subs=+quotes]
+--------------------------------------------------------------------------------
+$ **oc get pod -n eunomia-operator**
+NAME                               READY   STATUS    RESTARTS   AGE
+eunomia-operator-994b5d88d-ks6kw   1/1     Running   0          11m
+$ **oc logs -n eunomia-operator $(oc get pod -n eunomia-operator -o name)**
+{"level":"info","ts":1568215406.310659,"logger":"cmd","msg":"Go Version: go1.12.6"}
+{"level":"info","ts":1568215406.3106856,"logger":"cmd","msg":"Go OS/Arch: linux/amd64"}
+{"level":"info","ts":1568215406.3106892,"logger":"cmd","msg":"Version of operator-sdk: v0.8.1"}
+{"level":"info","ts":1568215406.3113315,"logger":"cmd","msg":"Templates initialized correctly"}
+{"level":"info","ts":1568215406.3116016,"logger":"leader","msg":"Trying to become the leader."}
+{"level":"info","ts":1568215406.4215379,"logger":"leader","msg":"No pre-existing lock was found."}
+{"level":"info","ts":1568215406.4294758,"logger":"leader","msg":"Became the leader."}
+{"level":"info","ts":1568215406.5258887,"logger":"cmd","msg":"Registering Components."}
+{"level":"info","ts":1568215406.526105,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"gitopsconfig-controller","source":"kind source: /, Kind="}
+{"level":"info","ts":1568215406.5262187,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"gitopsconfig-controller","source":"channel source: 0xc00096e7d0"}
+{"level":"info","ts":1568215406.5262578,"logger":"cmd","msg":"Starting the Web Server"}
+{"level":"info","ts":1568215406.5262656,"logger":"cmd","msg":"Starting the Cmd."}
+{"level":"info","ts":1568215406.6265268,"logger":"kubebuilder.controller","msg":"Starting Controller","controller":"gitopsconfig-controller"}
+{"level":"info","ts":1568215406.7266388,"logger":"kubebuilder.controller","msg":"Starting workers","controller":"gitopsconfig-controller","worker count":1}
+--------------------------------------------------------------------------------
+
+Create GitOps Repository
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Now we can set up eunomia to monitor a repo of configs, which in turn will apply configs to set up our cluster.
+Eunomia provides a custom resource kind called a `GitOpsConfig` to set up a monitor on a repository.
+You can examine the one we're going to use at [templates/cluster-gitops.yaml](templates/cluster-gitops.yaml).
+Let's use Applier to roll out the config.
+
+. Fork the link:https://github.com/redhat-cop/operationalizing-openshift-lab/[operationalizing-openshift-lab] repository into your own account.
++
+[subs=+quotes]
+--------------------------------------------------------------------------------
+$ *git clone https://github.com/_${GITHUB_ACCOUNT}_/operationalizing-openshift-lab*
```

Comment:
We should have students to an `export GITHUB_ACCOUNT=etsauer` since we are using this in a couple of places.